src/UKHO.ADDS.Clients.FileShareService.ReadWrite/FileShareReadWriteClient.cs

```diff
@@ -113,6 +182,24 @@ private HttpRequestMessage CreateHttpRequestMessage(Uri uri, BatchModel batchMod
             };
         }
 
+        private HttpRequestMessage CreateHttpRequestMessageForCommitBatch(Uri uri, IBatchHandle batchHandle)
+        {
+            return new HttpRequestMessage(HttpMethod.Put, uri)
+            {
+                Content = new StringContent(JsonCodec.Encode(batchHandle), Encoding.UTF8, "application/json")
```

Comment:
Can we add  it to ApiHeaderKeys.cs class as public const string ContentType = "application/json"; and use it accross.

Reply:
Created Variable ContentType in ApiHeaderKeys for application/json